api/models/utils.py

```diff
@@ -17,3 +17,8 @@ def save_to_db(self):
             db.session.rollback()
             db.session.flush()
             print(e)
+
+
+def insert_initial_values():
+    util = Utilities(pricing=0.00)
```

Comment:
Won't this will insert new entry every time server restarts and gets first request.

Reply:
Instead of autopopulating id, check for already present id and if not present them the same in the DB

Reply:
Done @yashLadha